paper-input-behavior.html

```diff
@@ -356,6 +356,15 @@
 
     attached: function() {
       this._updateAriaLabelledBy();
+
+      // These types have some default placeholder text; overlapping
```

Comment:
All of the date ones only work in Chrome, otherwise browsers default to `text`. The `file` inputs are a little different and inconsistent, but the `placeholder` attribute is always available to be used (and looks like an unfloated label), so I'd rather err on the "always float in this case" rather than try to split it on browsers. WDYT?